src/@commonsku/styles/Thermometer.tsx

```diff
@@ -39,19 +38,59 @@ const ProgressBar = styled.div<{
     position: absolute;
 `
 
+interface ThermometerLabelProps {
+    className?: string;
+    value: number;
+    label?: React.ReactNode | ((v: number) => React.ReactNode);
+}
+
+const ThermometerLabel = forwardRef<HTMLSpanElement, ThermometerLabelProps>((
+    { className, value, label }: ThermometerLabelProps, 
```

Comment:
no you don't need to specify the types for function params anymore

Reply:
I believe the types are inferred but there's no harm in leaving them in